src/build.sh

```diff
@@ -111,14 +124,87 @@ function kernel_build()
     statistics_manager "build" "$runtime"
   fi
 
+  pkg_info "$pkg"
+
   return "$ret"
 }
 
+# This function returns the makefile rule for creating a package, according to
+# the type of package required
+#
+# @pkg_type The type of package created
+#
```

Comment:
You have a return here (`echo`), please, document it.

Reply:
Hi @joaoseckler , do not forget to document the return value here (in this case, the echo value).